Topic 2 - Ex 1/src/RecentFileList.java

```diff
@@ -0,0 +1,30 @@
+import java.util.ArrayList;
+
+public class RecentFileList {

```

Comment:
I don't know if the bootcamp mentions it, but let's start using javadoc, just write /** and the IDE will autocomple, then desscribe what the class does